cmd/monaco/v2/legacy/integration_test_utils.go

```diff
@@ -168,24 +168,39 @@ func cleanupIntegrationTest(t *testing.T, fs afero.Fs, envFile, suffix string) {
 // e.g. my-config_1605258980000_Suffix
 
 func RunLegacyIntegrationWithCleanup(t *testing.T, configFolder, envFile, suffixTest string, testFunc func(fs afero.Fs)) {
-	envvars.InstallFakeEnvironment(map[string]string{
+	configFolder, _ = filepath.Abs(configFolder)
+	envFile, _ = filepath.Abs(envFile)
+
+	envvars.InstallFakeOsOverrideEnvironment(map[string]string{
 		"CONFIG_V1": "1",
```

Comment:
Why are we still working with old format config? Why not converting to the new format?

Reply:
Since we are still supporting the legacy mode, it makes sense
to test it via integration tests (also that is the reason they are located
in the `legacy` folder).

As soon as we drop this migration layer, we can drop those tests. 

The new integration tests will probably be just the converted ones.

Reply:
But wouldn't it make sense to convert these tests in v2 right away? We can run these in parallel. v2 will be out main version so we need to have integration tests for it.

Reply:
I will add this with their own PR, since I wanted to get the old ones working.
We probably need to adjust our framework around our integration tests a bit.